### PR TITLE
refactor(Colors): Update Color Palette definition.

### PR DESCRIPTION
**Pull request by jmpg93**
Hi there!

This PR is about updating the catalog to match what design had in mind: https://www.figma.com/file/OVN2YGEZZ0Gj7TXVoMblli/M%C3%ADstica-Catalog?node-id=1%3A6237

The new catalog uses different colors and have a new Colors tab.

https://user-images.githubusercontent.com/5618731/183922400-12fa1200-d5aa-4c25-8ef0-41163bf1e397.mp4

### CODE REVIEW

File: MisticaCatalog/Source/Catalog/CatalogRow.swift [hunk 1]
```diff
@@ -19,7 +19,6 @@ enum CatalogRow: Int, CaseIterable, Identifiable {
     case chips
     case carousel
     case feedbacks
-    case fonts
```
Comment (jmpg93) [inline MisticaCatalog/Source/Catalog/CatalogRow.swift:22]: This is now a tab

File: MisticaCatalog/Source/ColorsView.swift [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            .sorted { lItem, rItem in lItem.name < rItem.name }
+            .filter { element in
+                guard !searchText.isEmpty else { return true }
+                let name = element.name.lowercased()
+                let paletteName = (element.paletteName ?? "").lowercased()
+                let hex = element.color.hexString.lowercased()
+                let searchText = searchText.lowercased()
+                return name.contains(searchText) || paletteName.contains(searchText) || hex.contains(searchText)
+            }
+    }
```
Comment (jmpg93) [inline MisticaCatalog/Source/ColorsView.swift:69]: Mirroring magic to retrieve the colors.

File: Sources/MisticaCommon/Colors/BlauColorPalette.swift [hunk 3]
```diff
@@ -9,162 +9,166 @@
...
+    var errorLow = palette.blauRed10 | palette.white.withAlphaComponent(0.05)
+    var promoLow = palette.blauPurple10 | palette.white.withAlphaComponent(0.05)
+    var brandLow = palette.blauBlueSecondary10 | palette.white.withAlphaComponent(0.05)
+    var successHigh = palette.blauGreen70 | palette.blauGreen30
+    var warningHigh = palette.blauYellow70 | palette.blauYellow40
+    var errorHigh = palette.blauRed70 | palette.blauRed40
+    var promoHigh = palette.blauPurple | palette.blauPurple30
 }
 
-public enum BlauColorPalette {
```
Comment (jmpg93) [inline Sources/MisticaCommon/Colors/BlauColorPalette.swift:129]: As we going to make a breaking change. I've updated the color definition.
No color changes, just moved the static properties to properties.

**Comment by jmbrocal** [comment]
The new catalog is pretty cool!!!

But I see something weird in the color pallete. When the color is the same color as the background you cannot see the color XD.. Can we do something to improve this colors? Changing the background or adding a border to the circle?

WTDY?

**Review by jmbrocal — DISMISSED** [review]

**Comment by jmpg93** [comment]
> The new catalog is pretty cool!!!
> 
> But I see something weird in the color pallete. When the color is the same color as the background you cannot see the color XD.. Can we do something to improve this colors? Changing the background or adding a border to the circle?
> 
> WTDY?

Right, I will try to improve it.

**Comment by jmpg93** [comment]
> The new catalog is pretty cool!!!
> 
> But I see something weird in the color pallete. When the color is the same color as the background you cannot see the color XD.. Can we do something to improve this colors? Changing the background or adding a border to the circle?
> 
> WTDY?

Done!

image

**Review by jmbrocal — APPROVED** [review]

**Review by salavert — APPROVED** [review]
Really nice

**Pull request merged**